enioka_scan/src/main/java/com/enioka/scanner/sdk/zebraoss/ssi/SsiCommand.java

```diff
@@ -25,7 +25,7 @@ public enum SsiCommand {
     SSI_MGMT_COMMAND(0x80, SsiSource.BOTH, true, new RsmResponseParser()),
     SCANNER_INIT_COMMAND(0x90, SsiSource.HOST),
     SCANNER_INIT(0x91, SsiSource.DECODER, new ScannerInitParser()),
-    TEMP_COMMAND(0x93, SsiSource.HOST),
+    TEMP_COMMAND(0x93, SsiSource.BOTH, true, new GenericParser()), // Not documented, source of problems
```

Comment:
are we sure we receive a response for every TEMP command? When is that command sent by the device?

Reply:
It turns out that when scanning symbology-changing barcodes the scanner does emit this specific OPCODE, which causes our translator to request a parser for it, which did not exist and caused a crash due to unexpected packet. This change adds a default parser that does nothing but still "acknowledges" the packet to prevent a crash.